src/tools.ts

```diff
@@ -144,7 +144,7 @@ export const createKeyPair = () => {
   }
 }
 
-export interface JwtPayload {
+export interface JwtPayload extends Record<string, unknown> {
```

Comment:
Might be useful for people add custom props to the payload. I am not sure whether this is legal...